crates/environ/src/builtin.rs

```diff
@@ -104,6 +93,19 @@ macro_rules! foreach_builtin_function {
             /// TODO
             drop_cont_obj(vmctx: vmctx, contobj: pointer);
 
+            /// General-purpose allocation. Only used by typed-continuations
+            /// code at the moment.
+            allocate(vmctx: vmctx, size: i64, align: i64) -> pointer;
```

Comment:
I suggest we prefix our libcalls to disambiguate ourselves from the upcoming GC libcalls or future proposal libcalls.

```suggestion
            tc_allocate(vmctx: vmctx, size: i64, align: i64) -> pointer;
```

Reply:
I will bulk-rename all of them as a separate PR after this lands.